runtime/common/Cargo.toml

```diff
@@ -12,16 +12,16 @@ version = "0.0.0"
 targets = [ "x86_64-unknown-linux-gnu" ]
 
 [dependencies]
-codec = { workspace = true, default-features = false, features = [ "derive" ] }
-scale-info = { workspace = true, default-features = false, features = [ "derive" ] }
+codec = { workspace = true, features = [ "derive" ] }
+docify.workspace = true
+scale-info = { workspace = true, features = [ "derive" ] }
 
 # Substrate
-frame-support = { workspace = true, default-features = false }
-sp-runtime = { workspace = true, default-features = false }
-sp-std = { workspace = true, default-features = false }
+frame-support.workspace = true
+sp-runtime.workspace = true
 
-parachains-common = { workspace = true, default-features = false }
-polkadot-primitives = { workspace = true, default-features = false }
+parachains-common.workspace = true
+polkadot-primitives.workspace = true
 
```

Comment:
Couldn't resist doing a small refactor here making it consistent with the rest of the repo :)